dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/NetUtils.java

```diff
@@ -51,6 +48,8 @@ private NetUtils() {
 
     private static volatile String HOST_ADDRESS;
 
+    private static  String DOLPHIN_SCHEDULER_PREFERRED_NETWORK_INTERFACE = "dolphin.scheduler.network.interface.preferred";
+
```

Comment:
Hi,

Is it better to move `DOLPHIN_SCHEDULER_PREFERRED_NETWORK_INTERFACE` to `Constants.java`?

Reply:
Thanks for your suggestions, I have completed the changes.